tempest/run-tempest.sh

```diff
@@ -57,6 +59,16 @@ export NOSE_WITH_XUNIT=1
 export NOSE_XUNIT_FILE=tempest_xunit.xml
 
 cd /usr/share/openstack-tempest-juno/
+
+# Quick check using Javelin
+if javelin_check_resources; then
```

Comment:
and also you'll have to submit a patch to edeploy metadatas for J.1.0.0 release and exclude this file, if you choose this solution. We don't want to loose this file after an upgrade ;-)